### PR TITLE
helmsman: init at 3.3.0

### PR DESCRIPTION
**Pull request by Lynty**
###### Motivation for this change

Adds https://github.com/Praqma/helmsman, a useful tool which allows you to automate the deployment/management of your Helm charts from version controlled code.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/cluster/helmsman/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "Praqma";
+    repo = "helmsman";
+    rev = "v${version}";
+    sha256 = "0i7sg3iwxb07gjxcz6chpdcx3fqykzldmf7s1c9m02hkps910ca8";
+  };
+
+  goPackagePath = "github.com/Praqma/helmsman";
```
Comment (geistesk) [inline pkgs/applications/networking/cluster/helmsman/default.nix:16]: `goPackagePath` is required for legacy `buildGoPackage`, but no longer for `buildGoModule`.

File: pkgs/applications/networking/cluster/helmsman/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{ lib, buildGoModule, fetchFromGitHub, ... }:
+
+let version = "3.3.0"; in
```
Comment (geistesk) [inline pkgs/applications/networking/cluster/helmsman/default.nix:3]: This is somewhat unusual. You can add `rec` after `buildGoModule` and replace the `inherit` with a `version` definition.

**Review by oxzi — CHANGES_REQUESTED** [review]
Thanks a lot for your first contribution to the nixpkgs!

Perhaps you want to change the addressed lines.

File: pkgs/applications/networking/cluster/helmsman/default.nix (hunk 2 above)
Comment (Lynty) [reply pkgs/applications/networking/cluster/helmsman/default.nix]: Thank you for your review 😊

**Review by oxzi — APPROVED** [review]
LGTM

File: pkgs/applications/networking/cluster/helmsman/default.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  };
+
+  modSha256 = "19qdrrwmjc32nw8m0zi251z32wqj2d956wgd1dkcvx1x0n4p435g";
+
+  meta = {
+    description = "Helm Charts (k8s applications) as Code tool";
+    homepage = "https://github.com/Praqma/helmsman";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ lynty ];
+    platforms = lib.platforms.unix;
```
Comment (jonringer) [inline pkgs/applications/networking/cluster/helmsman/default.nix:21]: just to make it a little more concise
```suggestion
  meta = with lib; {
    description = "Helm Charts (k8s applications) as Code tool";
    homepage = "https://github.com/Praqma/helmsman";
    license = licenses.mit;
    maintainers = with maintainers; [ lynty ];
    platforms = platforms.unix;
```

**Review by jonringer — COMMENTED** [review]
otherwise LGTM

**Comment by jonringer** [comment]
@Lynty thanks again for opening your first PR. PR volume is super high, and hard to get to everyone

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 86328 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>1 package built:</summary><br>- helmsman</details>

**Pull request merged**